src/Views/TryInstallView.vala

```diff
@@ -83,9 +84,20 @@ public class TryInstallView : AbstractInstallerView {
         var next_button = new Gtk.Button.with_label (_("Install %s").printf (Utils.get_pretty_name ()));
         next_button.get_style_context ().add_class (Gtk.STYLE_CLASS_SUGGESTED_ACTION);
 
+        var shutdown_button = new Gtk.Button.with_label (_("Shut Down…"));
```

Comment:
Since this shuts down immediately and there is no confirmation (or other further action), this shouldn't have an ellipsis.